docs/scripts/filter.js

```diff
@@ -0,0 +1,24 @@
+function applyFilters(){
+    var allBoxes = document.getElementsByClassName("brand");
+    var allChecked = document.getElementsByClassName("brandAll");
+    var checked = [];
+    for (var i=0; i < allBoxes.length; i++){
```

Comment:
Wanna explain your logic here? Does the if check if someone clicked the click all filter. If thats the case does it need to be in the for loop? Because lets say it was true then in that case it would run the if statement every single time it looped. Which would be O(n^2) runtime. Not toooooo hot

Reply:
But yah this is a possibility feel free to explain to me if im misunderstanding!

Reply:
OHHHH, no ur completely right, ill fix that